LiteDB/LiteDB.csproj

```diff
@@ -36,18 +36,20 @@
   HAVE_APP_DOMAIN
   HAVE_PROCESS
   HAVE_ENVIRONMENT
+  HAVE_SHA1_MANAGED
+  HAVE_GET_CULTURE_INFO
   -->
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
     <DefineConstants>TRACE;DEBUG</DefineConstants>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)' == 'net45'">
-    <DefineConstants>NET40;HAVE_APP_DOMAIN;HAVE_PROCESS;HAVE_ENVIRONMENT;</DefineConstants>
+    <DefineConstants>HAVE_GET_CULTURE_INFO;HAVE_SHA1_MANAGED;HAVE_APP_DOMAIN;HAVE_PROCESS;HAVE_ENVIRONMENT;</DefineConstants>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <DefineConstants>NETSTANDARD20;</DefineConstants>

```

Comment:
I have removed NETSTANDARD20 and NET40 constants, while they weren't used in project.
But I also want to let you know, that you can use predefined NET45, NETFRAMEWORK, NETSTANDARD, NETSTANDARD1_3, NETSTANDARD2_0.